### ISSUE TITLE
UnicodeDecodeError: 'charmap' codec can't decode byte 0x90 in position 5824: character maps to <undefined>

### ISSUE THREAD

**Issue by barryjj** [Jonius7/SteamUI-OldGlory#19]
==============================
Running js_tweaker
Opening JS file and beautifying...
Writing beautified file... please do not close
Beautified file write finished
library.js changing to use tweaked JS.
library.js not found
~~~~~~~~~~
Traceback (most recent call last):
  File "js_tweaker.py", line 92, in modify_library
  File "c:\program files\0b coding\python38\lib\encodings\cp1252.py", line 23, in decode
UnicodeDecodeError: 'charmap' codec can't decode byte 0x90 in position 5824: character maps to <undefined>
None
~~~~~~~~~~
input(): lost sys.stdin

**Comment by barryjj** [comment]
That hardcoded path looks odd; I definitely do not have a 0b coding directory in C:\program files\ and quick scan of the readme for install doesn't mention anything of the sort that I can see.

This started happening back when the new steam update was pushed out a week or two ago -- I saw that there'd been activity and an update via the small fixes function, so I gave it another try, but I'm still hitting this path/code page error now.

**Comment by Jonius7** [comment]
There was a similar error that I fixed back in [5.5.1] (https://github.com/Jonius7/SteamUI-OldGlory/releases/tag/Release_5.5.1).

So I've had a look and made a quick fix that should fix the current issue. 

I can't guarantee a release right now, however, since the current code is in the middle of being reworked for `Release 6.0`. If I do, I'll reply to this thread again.
